public/app/app.config.js

```diff
@@ -0,0 +1,28 @@
+/*!
```

Comment:
@nicolaasmatthijs interesting that we're adding the license to every single file. Wouldn't it suffice to add it to:

1. The main file `app.module.js`
1. In the root directory (seems like we already have that at https://github.com/ets-berkeley-edu/collabosphere/blob/master/LICENSE)

Now, especially since we're using the `!` notation, I'm wondering how many times it'll show up in the concatenated and even minified file.